.github/workflows/build.yml

```
@@ -20,21 +20,25 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["8.0", "8.1"]
-                symfony: ["^5.4"]
+                symfony: ["^5.4", "^6.0"]
                 sylius: ["~1.11.0", "~1.12.0"]
```

```suggestion
                sylius: ["~1.11.10", "~1.12.0"]
```
We should probably change this as it requires 1.11.0 with PHP [8.1](https://github.com/Sylius/InvoicingPlugin/actions/runs/3392719526/jobs/5639232073#step:13:160) and 1.11.10 with [8.0](https://github.com/Sylius/InvoicingPlugin/actions/runs/3392719526/jobs/5639231592#step:13:161) 🤔